content/200-concepts/100-components/01-prisma-schema/06-relations/410-referential-actions/100-special-rules-for-referential-actions.mdx

```diff
@@ -11,7 +11,7 @@ Some databases have specific requirements that you should consider if you are us
 
 - Microsoft SQL Server doesn't allow cascading [referential actions](./) on a foreign key, if the relation chain causes a cycle or multiple cascade paths. If the referential actions on the foreign key are set to something other than `NO ACTION` (or `NoAction` if Prisma is managing referential integrity), the server will check for cycles or multiple cascade paths and return an error when executing the SQL.
 
-- with MongoDB, using referential actions in Prisma requires that for any data model with self-referential relations or cycles between three models, you must set the referential action of `NoAction` to prevent the referential action emulations from looping infinitely. Be aware that by default, the `referentialIntegrity = "prisma"` mode is used for MongoDB, which means that Prisma manages [referential integrity](/concepts/components/prisma-schema/relations/referential-integrity)
+- With MongoDB, using referential actions in Prisma requires that for any data model with self-referential relations or cycles between three models, you must set the referential action of `NoAction` to prevent the referential action emulations from looping infinitely. Be aware that by default, the `relationMode = "prisma"` mode is used for MongoDB, which means that Prisma manages [referential integrity](/concepts/components/prisma-schema/relations/referential-integrity). Note that the `relationMode` field was renamed in Prisma version 4.5.0, and was previously named `referentialIntegrity`.
```

Comment:
No need to mention this here as this is not something a user would configure:
```suggestion
- With MongoDB, using referential actions in Prisma requires that for any data model with self-referential relations or cycles between three models, you must set the referential action of `NoAction` to prevent the referential action emulations from looping infinitely. Be aware that by default, the `relationMode = "prisma"` mode is used for MongoDB, which means that Prisma manages [referential integrity](/concepts/components/prisma-schema/relations/referential-integrity).
```